aws-greengrass-testing-features/aws-greengrass-testing-features-cloudcomponent/src/main/resources/greengrass/features/cloudComponent.feature

```diff
@@ -39,3 +39,16 @@ Feature: Testing Cloud component in Greengrass
     And I deploy the Greengrass deployment configuration
     Then the Greengrass deployment is COMPLETED on the device after 180 seconds
     And the com.aws.HelloWorldMultiplatform log on the device contains the line "Hello World!" within 20 seconds
+
+  @CloudDeployment @IDT @OTFStable
```

Comment:
do not mark as IDT or clouddeployment

Reply:
Fixed in deb9dbb141753ed331184d937c302e73786c8a4d